pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
> Line 203 switched #ifdef BIT64 to #ifndef BIT64. The problem is in cmake we are defining BIT64 here. This change thus switches cmake build code path, changing our __int64 def on 64bit unix from long to long long.

Also, on 32bit unix it would be wrong -- defining __int64 to ```long```.